feature/home/presentation/viewmodel/src/main/java/com/chocolatecake/viewmodel/people/PeopleDetailsUiState.kt

```diff
@@ -0,0 +1,35 @@
+package com.chocolatecake.viewmodel.people
+
+import com.chocolatecake.viewmodel.common.model.MediaVerticalUIState
+
+
+data class PeopleDetailsUiState(
+    val peopleData :PeopleDataUiState =PeopleDataUiState(0,"","","","","","","") ,
```

Comment:
```suggestion
    val peopleData :PeopleDataUiState =PeopleDataUiState() ,
```
the ui state class must have already inital values